spec/factories/draft_work.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :draft_work do
+    sequence(:title) { |n| "#{n}人はプリキュア" }
+    media :tv
+    official_site_url 'http://example.com'
+    wikipedia_url 'http://example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.